### ISSUE TITLE
darwin_arm64 release

### ISSUE THREAD

**Issue by pablosichert** [hetznercloud/terraform-provider-hcloud#291]
Running `terraform init` on the `arm64` version for macOS currently yields

```
Error: Incompatible provider version

Provider registry.terraform.io/hetznercloud/hcloud v1.23.0 does not have a
package available for your current platform, darwin_arm64.

Provider releases are separate from Terraform CLI releases, so not all
providers are available for all platforms. Other versions of this provider may
have different platforms supported.
```

The workaround is to have separate installations (`arm64`/`x86_64`) of homebrew and use the `x86_64` version for now.

**Comment by LKaemmerling** [comment]
Hey @pablosichert,

thank you for your issue. Sorry for the short delay, the integrations team was on vacation. 

We are here limited by what terraform (or more generally go) supports. Currently, we build our provider against go 1.15 which does not support Apple Silicon yet. Support for this is planned/implemented with Go 1.16, which is expected to be released in Feb 2021. We will then update and provide binaries for darmin_arm64. As this is a normal maintenance task, i will close the issue. 

If you still have any problems or questions feel free to write under this issue or open up a new issue.

**Comment by LKaemmerling** [comment]
For our internal records:
We also need to wait for https://github.com/goreleaser/goreleaser/issues/1952
